### ISSUE TITLE
Hi author, can you provide a direct copy of the Waymo SOT dataset that has been processed? 

### ISSUE THREAD

**Issue by 2311762665** [Ghostish/Open3DSOT#30]
Hi author, can you provide a direct copy of the Waymo SOT dataset that has been processed? I am confused about which version of the data is available on the official website and the subsequent processing of the data.

**Comment by Ghostish** [comment]
Yes, we would like to do so. But Waymo dataset is too large and currently we cannot find a suitable cloud drive or open web server to store those files.

**Comment by WangJun-ZJUT** [comment]
Hi! friend, I have the same problem about the version of the data .  If you have make sure which version of the data is valid?  I'd appreciate it if you could help me.

**Comment by Ghostish** [comment]
Hi there.  We use the Perception dataset v1.2 from WOD. Actually, the version does not really matter because what we need from WOD is only the LiDAR data and bounding box annotations, which are identical throughout different versions.

In our experiments, we used the full training set to train our models.
For the testing data, we use the validation set from WOD.
Waymo provides 8 `tar` files ( validation_0000.tar - validation_0007.tar). In the paper, we only use 7 of them to do the evaluation.  We did not use `validation_0001.tar` in our experiments.

We found this setup is not clear in our main paper. We will update this detail in our arxiv revision in the future :)

Let me know if you still have problems
